CHANGELOG.md

```diff
@@ -19,6 +19,9 @@
 - Bump [ICS](https://github.com/cosmos/interchain-security) to
   [v4.1.0-lsm](https://github.com/cosmos/interchain-security/releases/tag/v4.1.0-lsm)
   ([\#3062](https://github.com/cosmos/gaia/pull/3062))
+- Bump [ICS](https://github.com/cosmos/interchain-security) to
```

Comment:
Why both ICS bumps? Shouldn't you delete one?

Reply:
v4.1.1-lsm is not state breaking, so it would be weird to put it in the state breaking section. It would also be weird to have v4.1.0-lsm in the state breaking and only v4.1.1-lsm as a dependency. I think like this is less confusing.